sdk/highlight-node/src/client.ts

```diff
@@ -332,10 +332,22 @@ export class Highlight {
 			span.setAttributes(metadata)
 		}
 		if (secureSessionId) {
-			span.setAttributes({ ['highlight.session_id']: secureSessionId })
+			span.setAttribute('highlight.session_id', secureSessionId)
```

Comment:
Well THAT'S a good catch.

Reply:
haha i think it's equivalent; just thought i'd simplify this